<ros2_control name="CurtisMotorControl" type="actuator">
  <hardware>
    <plugin>caddy_ai2_ros2_control_hardware_curtis_motor_driver/CurtisMotorDriver</plugin>
    <param name="can_interface">can0</param>
    <param name="watchdog_command">true</param>
    <param name="mode">0</param>
    <param name="wheel_diameter">0.5</param>
    <param name="max_rpm">4300.0</param>
    <param name="gear_ratio">16.0</param>
    <param name="max_v_auto">2.0</param>
    <param name="motor_direction">-1</param>
  </hardware>
  <joint name="curtis_motor">
    <command_interface name="throttle"/>
    <state_interface name="velocity"/>
    <state_interface name="current_rms"/>
    <state_interface name="battery_current"/>
    <state_interface name="voltage"/>
    <state_interface name="bdi_percentage"/>
    <state_interface name="motor_rpm"/>
    <state_interface name="temperature"/>
    <state_interface name="motor_temperature"/>
  </joint>
</ros2_control>